### ISSUE TITLE
Install Unsupported on arm64

### ISSUE THREAD

**Issue by kfrp** [edgedb/edgedb-cli#642]
Platform: Ubuntu 20.04
Version: 1.0.0-rc.4 (latest)
Architecture: `arm64`

Installation method: `curl --proto '=https' --tlsv1.2 -sSf https://sh.edgedb.com | sh`
Notes: AWS Graviton2 processor 

Output of `edgedb server _detect`:  n/a

**Comment by tailhook** [comment]
For now we don't support server on arm64. So we also don't build CLI tool (that will be unable to install the server), to avoid user confusion.

You can build just the CLI manually, though, if you have Rust/Cargo installed. It should be as easy as:
```
cargo install https://github.com/edgedb/edgedb-cli
```

Although, I agree that this is not convenient. We don't have any estimate for when arm64 will be supported.

Do you have any specific use case for arm64? It should be easy enough to run EdgeDB on x86_64 host and any other app on arm64 within Amazon cloud.

**Comment by kfrp** [comment]
Thanks for the reply. I switched to a x86_64 instance so it's not a problem for me, I just logged it here because there was no issue mentioning it.

**Comment by tailhook** [comment]
Let's keep this issue open, until we have the support.

**Comment by tiberiusferreira** [comment]
Noting that ARM support might also help: https://github.com/edgedb/edgedb-cli/issues/638

**Comment by D1no** [comment]
Just figured this out. Came originally here cause rust <-> cross system. Since there are 20 Mio m1 macs out there now and more than half of engineers use them: Would like to know if that is on the roadmap to keep local dev environments easy to set-up and predictable.

**Comment by elprans** [comment]
Native m1 builds are being worked on.  The biggest roadblock currently is lack of native Github actions runners, so we need to do some infra work first.

**Comment by D1no** [comment]
Understood. @elprans maybe you want to wait a bit then — github [has something](https://github.com/github/roadmap/issues/240) on the roadmap. I’d rather have you focus on the product instead of getting distracted by me here. Was just the first blocker I noticed. Running docker in compat mode for now.

**Comment by elprans** [comment]
Just to clarify: EdgeDB works via Rosetta, you don't need to use docker (just make sure your terminal has Rosetta enabled).

**Comment by D1no** [comment]
Yes just don’t want to install stuff on my / any machine 🤗 always work from inside containers.
